explorer/app/components/Project/components/Citation/citation.tsx

```diff
@@ -0,0 +1,49 @@
+// Core dependencies
+import { Typography } from "@mui/material";
+import React from "react";
+
+// App dependencies
+import { useConfig } from "app/hooks/useConfig";
+import { Stack } from "../../../common/Stack/Stack";
+import { ANCHOR_TARGET, Link } from "../../../Links/components/Link/link";
+import { Section } from "../Section/section";
+
+// Styles
+import { CitationLink } from "./citation.styles";
+
+interface Props {
+  citationPath: string;
+}
+
+export const Citation = ({ citationPath }: Props): JSX.Element => {
+  const { datasources } = useConfig();
+  const { url } = datasources;
+  const citationLink = `${url}${citationPath}`;
+  const [root, projectId] = citationPath.split(/\/(?!.*\/)/);
+  const CitationLinkLabel = (
```

Comment:
CitationLinkLabel is built up to ensure the paths are kept intact as the viewport decreases in size.